react-dnd/react-dnd-html5-backend.d.ts

```diff
@@ -6,7 +6,11 @@
 ///<reference path='./react-dnd.d.ts' />
 
 declare module "react-dnd-html5-backend" {
-  export enum NativeTypes { FILE, URL, TEXT }
+  export var NativeTypes: {
+    FILE: string,
+    URL: string,
+    TEXT: string
+  };
```

Comment:
Looking at the [source](https://github.com/gaearon/react-dnd-html5-backend/blob/master/src/NativeTypes.js), I think it may be more accurate to type this using a [namespace](https://www.typescriptlang.org/docs/handbook/namespaces.html):

```ts
export namespace NativeTypes {
    export const FILE: string;
    ...
}
```